tests/manifests/controller_definitions.yaml

```diff
@@ -744,3 +744,773 @@ controllerDefinitions:
           - configMap:
               name: numaflow-controller-config
             name: controller-config-volume
+
+- version: "0.01" # "numaio-rc-0.01"
```

Comment:
I wonder if we may want to name these by their image names plus tags? I didn't expect Derek to create an image named "numaflow-rc:v0.01" (different semantic versioning scheme than "numaflow" image)